Modules/@babylonjs/react-native-basekit/README.md

```diff
@@ -0,0 +1,120 @@
+# Babylon React Native Basekit
+
+## Usage
+
+This quick overview will help you understand the constructs provided by Babylon React Native and how to use them in a React Native application.
+
+### Dependencies
+
+This package has several **peer dependencies**. If these dependencies are unmet, `npm install` will emit warnings. Be sure to add these dependencies to your project.
+
+### Installation Configuration
+
+Downloaded dependencies are cache by default in directory specifiec by `npm_config_cache`. It's possible to specify a new cache directory by setting `npm_config_babylon_binary_cache` environment variable.
+
+Also, it's possible to set the source of these downloaded dependencies. By default, it's using main Github repo releases: `https://github.com/BabylonJS/BabylonReactNative/releases/download`.
+Other possibilities are:
+- By command argument for `npm install` : `--brn-binary-site=https://mysite.com/downloads`
+- By environment variable `export npm_config_brn_binary_site=https://mysite.com/downloads`
+- By configuration in package.json: `nodeBabylonConfig: { binarySite: 'https://mysite.com/downloads'}`
```

Comment:
Where in the package.json? Like this is a top level json element or something?

Reply:
I've taken the code from node-sass but I can't find documentation describing in details how it works. I'm removing it for now.